python/lsst/shapelet/radialProfile/radialProfileContinued.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+from .radialProfile import RadialProfile
+
+from lsst.utils import continueClass
+
+__all__ = []
+
+@continueClass
+class RadialProfile:
+    def evaluate(self, r):
+        if isinstance(r, np.ndarray):
+            return self._evaluate(r.ravel()).reshape(r.shape)
```

Comment:
Interesting that Python accepts n-dimensional data and C++ does not. But presumably we rely on that. Pity the C++ isn't more flexible so we could skip this. I realize a lambda function could handle this, but that would take significantly more code than we have here.